app/views/shared/_dialog.html.erb

```diff
@@ -0,0 +1,38 @@
+<div id="<%= id %>" class="sci-dialog modal fade" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+        <h4 class="modal-title"><%= title %></h4>
+      </div>
+      <div class="modal-body">
+        <div class="sci-dialog__icon <%= type %>">
+          <% case type %>
+          <% when 'error', 'warning' %>
+            <i class="fas fa-exclamation-triangle"></i>
+          <% when 'info' %>
+            <i class="fas fa-info"></i>
+          <% when 'success' %>
+            <i class="fas fa-check-circle"></i>
+          <% else %>
+            <i class="fas fa-info"></i>
+          <% end %>
+        </div>
+        <div class="sci-dialog__body">
+          <%= body.html_safe %>
```

Comment:
Why html_safe?

Reply:
Because in this case, the translation includes some bold tags.

Reply:
Hm does `_html` suffix in I18n fix this?

Reply:
Ah, didn't know that, yes it works without .html_safe. Fix pushed.